test/manifold_test.cpp

```diff
@@ -784,3 +784,56 @@ TEST(Manifold, EmptyHull) {
       {0, 0, 0}, {1, 0, 0}, {0, 1, 0}, {1, 1, 0}};
   EXPECT_TRUE(Manifold::Hull(coplanar).IsEmpty());
 }
+
+TEST(Manifold, ConvexConvexMinkowski) {
+  bool oldDeterministic = ManifoldParams().deterministic;
+  ManifoldParams().deterministic = true;
+  Manifold sphere = Manifold::Sphere(0.1, 20);
+  Manifold cube = Manifold::Cube({2.0, 2.0, 2.0});
+  Manifold sum = cube.MinkowskiSum(sphere);
+  EXPECT_FLOAT_EQ(sum.GetProperties().volume, 10.589364f);
```

Comment:
I believe for a cube and sphere we can calculate the analytical volume and surface area pretty easily, right? Let's do that and check both, along with the genus. That will do a good job of catching any internal slivers.

Reply:
You'll probably need to use `EXPECT_NEAR` instead though.

Reply:
True, though the sphere is pretty far from a perfect sphere's volume, so the "analytic" version ends up with an epsilon like:
```c++
  EXPECT_NEAR(sum.GetProperties().volume,
              powf(cubeWidth + (2 * offsetRadius), 3.f), 0.06f); // Big Epsilon from Imprecise Sphere Volume
```

Reply:
wonder how much better can we get if we increase the number of faces in the sphere.

Reply:
Also, are the points in our sphere meshes always inside the perfect sphere? Is it possible to change it such that it minimizes the volumetric difference instead? (just curious, not sure if it is a good idea)

Reply:
See the comment below - the issue isn't the sphere resolution, but the analytic result.